series-tmdb.en.yaml

```diff
@@ -913,6 +913,21 @@ entries:
       - season: 1
         anilist-id: 5114
 
+  - title: "Full Metal Panic!"
+    guid: plex://show/5d9c086fcb3ffa001f1b5059
+    # imdb: https://www.imdb.com/title/tt1355642/
+    # tmdb: https://www.themoviedb.org/tv/31911
+    # tvdb: https://www.thetvdb.com/dereferrer/series/78914
```

Comment:
The correct links for all 3 is the following:
```yaml
  # imdb: https://www.imdb.com/title/tt0328739/
  # tmdb: https://www.themoviedb.org/tv/39379
  # tvdb: https://www.thetvdb.com/dereferrer/series/78914
```

Reply:
fixed